### PR TITLE
[CARBONDATA-3133] Update the document to add spark 2.3.2 and datamap mv compiling method.

### PR DESCRIPTION
**Pull request by Jonathan-Wei**
Update the document to add spark 2.3.2 and datamap mv compiling method.

Update file:
https://github.com/apache/carbondata/tree/master/build/README.md

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [x] Any interfaces changed?
 No
 - [x] Any backward compatibility impacted?
 No
 - [x] Document update required?
Yes
 - [x] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [x] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
No

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1545/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9805/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1757/

File: build/README.md [hunk 1]
```diff
@@ -29,10 +29,40 @@ Build with different supported versions of Spark, by default using Spark 2.2.1 t
...
+mvn -DskipTests -Pspark-2.3 -Dspark.version=2.3.2 clean package
 ```
 
 Note: If you are working in Windows environment, remember to add `-Pwindows` while building the project.
 
+## MV Feature Build
+Add mv module and sourceDirectory to the spark profile corresponding to the parent pom.xml file and recompile.
+The compile command is the same as the command in the previous section
+```
+    <profile>
```
Comment (xuchuanyin) [inline build/README.md]: Do we really need this?
Currently we can use `-Pmv` to include the MV feature while compiling.

File: build/README.md [hunk 2]
```diff
@@ -29,10 +29,40 @@ Build with different supported versions of Spark, by default using Spark 2.2.1 t
 ```
 mvn -DskipTests -Pspark-2.1 -Dspark.version=2.1.0 clean package
 mvn -DskipTests -Pspark-2.2 -Dspark.version=2.2.1 clean package
+mvn -DskipTests -Pspark-2.3 -Dspark.version=2.3.2 clean package
 ```
 
 Note: If you are working in Windows environment, remember to add `-Pwindows` while building the project.
 
+## MV Feature Build
```
Comment (xubo245) [inline build/README.md]: no need so lone describe, only add `-Pmv` , for example:If you want to use MV, remember to add `-Pmv ` while building the project.

Comment (xubo245) [reply build/README.md]: in the parent pom.xml, there is already mv profile.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1548/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1760/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9808/

**Comment by xubo245** [comment]
LGTM

**Comment by xuchuanyin** [comment]
LGTM

**Comment by zzcclp** [comment]
LGTM

**Comment by xuchuanyin** [comment]
Merged. Thanks for  your contribution 👍

**Pull request closed without merging**